src/test/kotlin/no/nav/dagpenger/regel/grunnlag/ApplicationTopologyTest.kt

```diff
@@ -0,0 +1,79 @@
+package no.nav.dagpenger.regel.grunnlag
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import io.kotest.matchers.collections.shouldContain
+import io.kotest.matchers.ints.shouldBeExactly
+import io.mockk.mockk
+import no.nav.dagpenger.regel.grunnlag.RapidGrunnlag.Companion.BEREGNINGSDATO
+import no.nav.dagpenger.regel.grunnlag.RapidGrunnlag.Companion.GRUNNLAG
+import no.nav.dagpenger.regel.grunnlag.RapidGrunnlag.Companion.INNTEKT
+import no.nav.helse.rapids_rivers.InMemoryRapid
+import no.nav.helse.rapids_rivers.inMemoryRapid
+import org.intellij.lang.annotations.Language
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+class ApplicationTopologyTest {
+    private val instrumentation = mockk<GrunnlagInstrumentation>(relaxed = true)
+    private lateinit var rapid: InMemoryRapid
```

Comment:
InMemoryRapid er deprecated

Bruk `TestRapid` i stedet for - se feks https://github.com/navikt/dp-oppslag-arbeidsforhold/blob/master/src/test/kotlin/no/nav/dagpenger/oppslag/arbeidsforhold/ApplicationTopologyTest.kt#L29-L78

Reply:
vi hadde litt problemer med å få tak i den (selv om vi skal bruke nyeste versjon av rapid-rivers??), men kan se på det igjen

Reply:
vi hadde ikke merga master med nyeste versjon, så vi brukte ikke nyeste versjon... 🤦‍♀️